report/renderer/performance-category-renderer.js

```diff
@@ -167,6 +167,24 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
     return 'diagnostic';
   }
 
+  /**
+   * @param {LH.ReportResult.AuditRef[]} metrics
+   */
+  _reorderMetrics(metrics) {
+    const col1 = metrics
+      .splice(0, Math.floor(metrics.length / 2))
+      .reverse();
+    const col2 = metrics.reverse();
```

Comment:
specifically, I was wondering why this line wasn't a mistake, thinking it would duplicate the metrics. But the array was reduced in length in the prior line.